### ISSUE TITLE
Missing Landmarks that are shown by JAWS/NVDA

### ISSUE THREAD

**Issue by wilsmic2** [skipto/extension#3]
This is Mikey Wilson from the University of Washington's ATX team. We recently ran into an issue with the SkipTo extension where it was not recognizing all of the landmarks on a page, whereas JAWS/NVDA is able to.

Specifically, below are the landmark structures for each option:
NVDA:
- Form
  -  Session events; complementary
  - Main

JAWS:
- Banner
- Session events Complementary information
  -  Search
- Main

SkipTo:
- Main
- Search
- Complementary: session events

From these, we can see that the SkipTo extension is not recognizing the Banner or Form landmarks that NVDA/JAWS does.

We have created a recording of this issue so that you can see the website that we tested on as well as parts of the DOM, which you can find at this link:
https://www.dropbox.com/s/o77u0t4p2g8kfz9/skipto.mp4?dl=0

Unfortunately we cannot give access to the website that we tested on as it is internal to UW, but if you would like me to get you the code from the DOM to help debug then please let me know.
